src/content/community/releases/_index.en.md

```diff
@@ -11,6 +11,8 @@ weight = 40
   environment which may forbid certain configurations in a disconnected Azure installation.
 * `subctl` is now built for ARM Macs (Darwin arm64).
 * `subctl show versions` now shows the versions of the metrics proxy and plugin syncer components.
+* Fixes an issue with OVNKubernetes where routes could be accidently deleted when the cluster goes down and comes back
```

Comment:
A suggestion:

Fixes an issue with OVNKubernetes CNI where routes could be accidentally deleted during cluster restart, or upgrade scenarios.

Reply:
“Fixed”, please (I can’t remember where we had the discussion, but we agreed on past tense for release note entries phrased as changes, such as “Added”, “Fixed”, “Removed”).